about.html

```diff
@@ -0,0 +1,122 @@
+      </section>
+      <nav>
+        <a href="index.html"
+          ><img class="logo" src="./images/cc.logo.white.svg" alt="cc-logo`"
+        /></a>
+        <div>
+          <ul class="nav-menu">
+            <li><a class="link" href="about.html">About</a></li>
+            <li><a class="link" href="#">Program</a></li>
+            <li><a class="link" href="#">Join</a></li>
+            <li><a class="link" href="#">Sponsor</a></li>
+            <li><a class="link" href="#">News</a></li>
+          </ul>
+        </div>
+        <button>CC Campaign</button>
+        <div class="hamburger-menu">
+          <i class="fa-solid fa-bars"></i>
+          <i class="fa-solid fa-xmark"></i>
+        </div>
+      </nav>
+
+      <div class="hero">
+        <p class="hero-hello">"Hello Sharing World"</p>
+        <h1>Creative Commons Global Summit 2015</h1>
+        <p class="hero-p">
+          Creative Commons is a non-profit organization that enables the sharing
+          and use of creativity and knowledge through free legal tools.
+        </p>
+        <p class="hero-p-bold">
+          Please contact us per Email for any further questions about CC Global
+          Summit 2015
+        </p>
+        <p class="hero-national">
+          <a class="hero-mail" href="mailto:ccsummit2015@cckorea.org"
+            >ccsummit2015@cckorea.org</a
+          >
+        </p>
+      </div>
+    </header>
+
+    <main>
+      <h2>Creative Summons Global Summit 2015 Logo</h2>
+      <div class="main-underline"></div>
+      <p>
+        The Logo of CC Global summit 2015 was decided through the logo
+        competition from 8, June to 7, July
+      </p>
+      <div class="main-img">
+        <img src="./images/CC_Global_Summit_2015.png" alt="logo" />
+      </div>
+      <hr />
+    </main>
+
+    <section class="summit-experience">
+      <h2>See the past CC Global Summits</h2>
+      <div class="main-underline"></div>
+      <p>
+        Take a look at the last two CC Global Summits which took place in 2013
+        in Warsaw, Poland and in 2014 in Marrakech, Morocco.
+      </p>
+      <div class="section-img">
+        <div class="section-summit-img">
+          <h3>2013</h3>
+          <p>The 2013 CC Global Summit was held in Warsaw.</p>
+        </div>
+        <div class="section-summit-img2">
+          <h3>2015</h3>
+          <p>CC Global Summit 2014 in Buernos Aires</p>
+        </div>
+      </div>
+    </section>
+
+    <section>
+      <div class="partner">
+        <p>Partner</p>
+        <div class="partner-underline"></div>
+        <div class="partner-logos">
+          <i class="fa-brands fa-google"></i>
+          <i class="fa-brands fa-firefox"></i>
+          <i class="fa-brands fa-airbnb"></i>
+          <i class="fa-brands fa-yahoo"></i>
+          <i class="fa-brands fa-reddit"></i>
+        </div>
+      </div>
+    </section>
+
+    <footer>
+      <div class="footer">
+        <img src="./images/cc.logo.white.svg" alt="" />
```

Comment:
Kindly, I suggest that for accessibility every time you use an "IMG" you indicate an appropriate "ALT" to describe the content of that image

Reply:
Done. Thank you for your reviews